src/Modules/ModParser.lua

```diff
@@ -1657,7 +1657,7 @@ local specialModList = {
 	["warcries grant (%d+) rage per (%d+) power if you have less than (%d+) rage"] = {
 		flag("Condition:CanGainRage"),
 	},
-	["exerted attacks deal (%d+)%% more damage if a warcry sacrificed rage recently"] = function(num) return { mod("ExertIncrease", "MORE", num, nil, ModFlag.Attack, 0) } end,
+	["exerted attacks deal (%d+)%% more attack damage if a warcry sacrificed rage recently"] = function(num) return { mod("ExertAttackIncrease", "MORE", num, nil, ModFlag.Attack, 0) } end,
```

Comment:
Why don't we change the old modifier name `ExertIncrease` to the new one `ExertAttackIncrease` and change the mod-parse line to:
```
["exerted attacks deal (%d+)%% more ?a?t?t?a?c?k? damage if a warcry sacrificed rage recently"]
```
This prevents the need for copying all the calculations for a new-named mod that is basically the old mod and provides legacy compatibility.